variable/include/scipp/variable/creation.h

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Simon Heybrock
+#pragma once
+#include <optional>
+
+#include "scipp-variable_export.h"
+#include "scipp/variable/variable.h"
+
+namespace scipp::variable {
+
+[[nodiscard]] SCIPP_VARIABLE_EXPORT Variable
+empty_like(const VariableConstView &prototype,
```

Comment:
I'm guessing this will be a candidate to expose to python, but that will be done at some later time https://numpy.org/doc/stable/reference/generated/numpy.empty_like.html

Reply:
👍